### ISSUE TITLE
Printing context on arm linux seems to run 'finish'

### ISSUE THREAD

**Issue by Grazfather** [hugsy/gef#74]
```
0x0004a519    <do_report+44>     bne.n   0x4a51e <do_report+50>
---------------------------------------------[threads]----
[#0] Id 1, Name: "test-prog", stopped
-----------------------------------------------[trace]----
[#0] RetAddr: 0x4a50c, Name: do_report()
[#1] RetAddr: 0x288a8, Name: xxx()
[#2] RetAddr: 0x316b2, Name: jjj.part()
[#3] RetAddr: 0x31af0, Name: rrr()
[#4] RetAddr: 0x2d12a, Name: ttt.part()
[#5] RetAddr: 0x30c6c, Name: fff()
[#6] RetAddr: 0x27706, Name: qqq()
[#7] RetAddr: 0x69a3c, Name: ttt()
[#8] RetAddr: 0x10112, Name: main()
---------------------------------------------------------------
gef> finish
"finish" not meaningful in the outermost frame.
gef> p/x $pc
$1 = 0x10112 # Note that this is the address at the lowest stack frame
```

**Comment by Grazfather** [comment]
By playing around with the layout I was able to narrow it down to the trace layout

**Comment by Grazfather** [comment]
Issue seems to be that when  gef prints the trace it isn't properly returning to the right frame after it's done :)

**Comment by Grazfather** [comment]
Fixed in #75
